tools/local-network/local_network.py

```diff
@@ -30,14 +30,14 @@
 PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
 MOB_RELEASE = os.getenv('MOB_RELEASE', '1')
 CARGO_FLAGS = '--release'
-TARGET_DIR = 'target/release'
-WORK_DIR =  os.path.join(PROJECT_DIR, TARGET_DIR, 'mc-local-network')
+TARGET_DIR = os.path.join(os.getenv('CARGO_TARGET_DIR', os.path.join(PROJECT_DIR, 'target')), 'release')
+WORK_DIR =  os.path.join(TARGET_DIR, 'mc-local-network')
```

Comment:
Optional/Equivalent: Extract a var for the base target dir, and use conditional expressions to replace the `if MOB_RELEASE == '0':` block below.

```suggestion
MOB_RELEASE = os.getenv('MOB_RELEASE', '1') == '1'
CARGO_FLAGS = '--release' if MOB_RELEASE else ''
BASE_TARGET_DIR = os.getenv('CARGO_TARGET_DIR', os.path.join(PROJECT_DIR, 'target'))
TARGET_DIR = os.path.join(BASE_TARGET_DIR, 'release' if MOB_RELEASE else 'debug')
```

Reply:
Did a similar and slightly different approach, the `if MOB_RELEASE` will always evaluate to true
```
>>> foo = "0"
>>> if foo: print("Toasty!!!")
... 
Toasty!!!
```

Reply:
Note that my suggestion changes the definition to be a proper boolean